README.md

```diff
@@ -41,39 +41,49 @@ This is the dependency you need:
 </dependency>
 ```
  
-Then, you implement `com.yegor256.rpm.Storage` interface.
+Then, use one of the existing `com.artipie.asto.Storage` implementations to create the `Storage`,
```

Comment:
@olenagerasimova It looks like something is missing in this sentence or a "." is missing.